destination_test.go

```diff
@@ -213,8 +143,7 @@ func TestDestinationPluginAdapter_Run_WriteFallback(t *testing.T) {
 	}()
 
 	for i := 0; i < 2; i++ {
-		// write the same record twice to check if WriteAsync is skipped the
-		// second time
+		// write the same record twice
```

Comment:
Agreed, it doesn't hurt writing 2 records. I'm simply wondering if they need to be the same (because we are testing XYZ which is triggered by identical records).

Reply:
Oh no, not at all, I can actually remove that comment because it may create more questions than answers 🙂